docs/core-concepts/libraries/libraries-with-cli.md

```diff
@@ -4,94 +4,83 @@ By [Phillip Carter](https://github.com/cartermp)
 
 **Some details are subject to change as the toolchain evolves.**
 
-Targeting .NET Core for libraries can be done entirely with the .NET CLI tools, which are a foundational set of tools used by Visual Studio, ASP.NET, and other technologies.  They provide an efficient and low-level experience that works across any supported OS.  You can still build libraries with Visual Studio, and if that is your preferred experience then you should [refer to the Visual Studio guide](libraries-with-vs.md).  This document will focus on using the CLI tools directly.
+Targeting .NET Core for libraries can be done entirely with the .NET CLI tools, which are a foundational set of tools used by Visual Studio and ASP.NET Core.  They provide an efficient and low-level experience that works across any supported OS.  You can still build libraries with Visual Studio, and if that is your preferred experience then you should [refer to the Visual Studio guide](libraries-with-vs.md).  This document will focus on using the CLI tools directly.
```

Comment:
remove the refer verb from the link text:
refer to the [Visual Studio guide](libraries-with-vs.md)